cherryontech/JourneyWise
# worked on JW-44

## Pull Request

### Description
Worked on the security display with a modal and included the checkbox 

### Related Issue
https://marisolsaumi.atlassian.net/browse/JW-44
https://marisolsaumi.atlassian.net/browse/JW-61
.

### Description of Changes
* I have created a security notice with a modal  
* The user can read the security notice and approve it . 

### Checklist
- [x ] I have read the CONTRIBUTING.md document.
- [ x] My code follows the code style of this project.
- [x ] My changes generate no new warnings or errors.
- [ x] I have updated the documentation or comments where necessary.
- [ x] All tests pass locally.

### Testing Instructions
* clone the repostiorty and click the button where it shows you the security notice and click the check box to see the button. 


### Screenshots (if applicable)
<img width="522" alt="Screenshot 2024-05-04 at 03 35 17" src="https://github.com/cherryontech/JourneyWise/assets/77724559/ccce31cf-7077-4f5c-9c50-4464b0857c0b">

### Gotchas/Learnings
Still leaning tailwind 

### Additional Notes (Optional)
* Include any other relevant information for reviewers.
